docs/user/spark/sparksql.rst

```diff
@@ -279,3 +281,114 @@ write to a distributed file system like HDFS.
 
     For this to work, the Data Frame should have a geometry field, meaning its schema should have a ``StructField`` that
     is one of the JTS geometry types provided by GeoMesa.
+
+Using GeoMesa SparkSQL with Apache Sedona
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+GeoMesa SparkSQL can work seamlessly with `Apache Sedona <http://sedona.apache.org/>`_. You can enable this feature by
+specifying ``geomesa.use.sedona`` system property to ``true`` when submitting your Spark job. Please make sure that
+Apache Sedona JAR ``sedona-python-adapter-${spark-version}_${scala-version}-${sedona-version}.jar`` is submitted with
+``--jars`` option:
+
+.. code-block:: shell
+
+   spark-submit --conf "spark.driver.extraJavaOptions=-Dgeomesa.use.sedona=true" \
+                --conf "spark.executor.extraJavaOptions=-Dgeomesa.use.sedona=true" \
```

Comment:
this seems good enough for a first pass, but I wonder if it would it be possible to detect sedona on the classpath and just enable it automatically if it's found?

Reply:
Enabling Sedona support silently could be a breaking change -- it will change the serde of geometry UDTs, which will break existing pyspark apps.

Reply:
maybe we could have a flag to disable it, but enable it by default if it's available. it seems unlikely someone would have sedona on the classpath but not want to use it? unless someone was using geomesa and sedona separately, but it seems like they would conflict.

Reply:
Maybe you are right. I'll enable it when `org.apache.spark.sql.sedona_sql.UDT.GeometryUDT` is available, and `geomesa.use.sedona` system property is not explicitly set to `false`.

Reply:
awesome!